packages/schemas/src/barcodes/propPanel.ts

```diff
@@ -181,7 +196,14 @@ export const getPropPanelByBarcodeType = (barcodeType: string): PropPanel<Barcod
         ],
       },
       ...(barcodeHasText
-        ? { textColor: { title: i18n('schemas.textColor'), type: 'string', widget: 'color' } }
+        ? {
+            textColor: { title: i18n('schemas.textColor'), type: 'string', widget: 'color' },
+            includetext: {
+              title: i18n('schemas.barcodes.includetext'),
+              type: 'boolean',
+              widget: 'switch',
```

Comment:
I changed it to a switch when placed next to a color input because I preferred the look. It's just a matter of personal preference.

![CleanShot 2023-12-29 at 13 21 30@2x](https://github.com/pdfme/pdfme/assets/24843808/19a52cad-e126-43d5-b159-f88297319672)

↓

![CleanShot 2023-12-29 at 13 21 01@2x](https://github.com/pdfme/pdfme/assets/24843808/7a74e877-4159-430c-b568-e1a66295f5a4)